program_transformers/src/bubblegum/db.rs

```diff
@@ -435,9 +434,7 @@ where
                     asset_data::Column::ChainData,
                     asset_data::Column::MetadataUrl,
                     asset_data::Column::MetadataMutability,
-                    // Don't update asset_data::Column::Metadata if it already exists.  Even if we
-                    // are indexing `update_metadata`` and there's a new URI, the new background
-                    // task will overwrite it.
+                    asset_data::Column::Metadata,
```

Comment:
Metadata should not be updated as it would over write what is potentially valid. Still enqueue a download job and if it is the same it will override it.

Metadata should only be set to proccesisng on insert.